net/instaweb/rewriter/critical_css_loader.js

```diff
@@ -39,11 +39,16 @@ pagespeed.CriticalCssLoader.addAllStyles = function() {
 
   var elements = document.getElementsByClassName('psa_add_styles');
 
+  
   for (var i = 0, e; e = elements[i]; ++i) {
     if (e.nodeName != 'NOSCRIPT') { continue; }
     var div = document.createElement('div');
     div.innerHTML = e.textContent;
-    document.body.appendChild(div);
+    var children = div.childNodes;
+    for (var v = 0; v < children.length; ++v) {
+      children[v].removeAttribute('id');
+    }  
+  document.body.appendChild(div);
```

Comment:
nit: indent

Reply:
The change you requested is to indent the line
  `document.body.appendChild(div);` ?

Reply:
yes, that would be great. But we may also want to recurse? So not only remove the `id` attribute from the direct children of the element, but also the children of the children, and so on?

Reply:
But childNode don´t get all childs and sub-childs nodes?

Reply:
Oh in that case you’re right this is OK.

Reply:
Sorry about the noise!